src/builtins/core/duration/normalized.rs

```diff
@@ -749,7 +749,7 @@ impl InternalDurationRecord {
         let beyond_day_span = rounded_time.checked_add(day_span.negate().0)?;
         // 12. If TimeDurationSign(beyondDaySpan) ≠ -sign, then
         let (expanded, day_delta, rounded_time, nudge_ns) =
-            if beyond_day_span.sign() != sign.negate() {
```

Comment:
nit: can you add the test case from the issue to `src/builtins/core/duration/tests.rs`

Ideally, when fixes go in, we've preferred to also add a corresponding Rust test so any possible regression is caught by `cargo test`

Reply:
Done in [fbf46d7](https://github.com/boa-dev/temporal/pull/647/commits/fbf46d759f4ff94d54ae808ea3332c0caa72f4f3)